wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -702,6 +704,14 @@ impl CommandHandler {
         ConsoleCommand::Print(status_text.to_owned())
     }
 
+    pub fn new_tx_submitted_command(new_tx: NewTransaction) -> ConsoleCommand {
+        let status_text = format!(
+            "The transaction was submitted successfully with ID {}",
```

Comment:
Can you make the id go into a new line, to simplify it for parsers to find it?

Reply:
fixed